thetaPC/dnd-one-shot
# feat(20136701, 20136714): Playable classes.

[Ticket 20136701](https://github.com/users/thetaPC/projects/1/views/1?pane=issue&itemId=20136701), [Ticket 20136714](https://github.com/users/thetaPC/projects/1/views/1?pane=issue&itemId=20136714)

How to test:
1. Run `npm install`
2. Run project
3. Verify that you can see the following classes: Barbarian, Fighter, and Monk

Screenshot:
![Screen Shot 2023-02-10 at 7 57 23 PM](https://user-images.githubusercontent.com/13530427/218238006-f52394ae-1b3c-491f-8524-dba45647f1b0.png)
